### PR TITLE
Update to docs/OAP-User-Guide.md

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

## What changes were proposed in this pull request?

Editorial review and update.

## How was this patch tested?

Reviewed rendered HTML

### CODE REVIEW

**Comment by zhixingheyi-tian** [comment]
@intelkevinputnam 

   Hi Kevin, PR#1198 merged caused a conflict.
   Please have a check

Thanks

**Comment by intelkevinputnam** [comment]
@zhixingheyi-tian Should be fixed now.

**Comment by zhixingheyi-tian** [comment]
@intelkevinputnam 
   Thanks very much,

 cc @HongW2019

File: docs/OAP-User-Guide.md [hunk 1]
```diff
@@ -82,139 +100,163 @@ After a successful OAP integration, you can use OAP SQL DDL to manage table inde
...
-. $SPARK_HOME/sbin/start-thriftserver.sh
-```
-Step3. Use Beeline and connect to the Thrift JDBC server using the following command, replacing the hostname (mythriftserver) with your own Thrift Server hostname.
+2. Launch Spark ***ThriftServer***
 
-```
-./beeline -u jdbc:hive2://mythriftserver:10000       
-```
-After the connection is established, execute the following command to check the metastore is initialized correctly.
+   Launch Spark Thift Server, and use the Beeline command line tool to connect to the Thrift Server to execute DDL or DML operations. The data cache will automatically take effect for Parquet or ORC file sources.
```
Comment (HongW2019) [inline docs/OAP-User-Guide.md:152]: A little typo here "Thrift Server"

File: docs/OAP-User-Guide.md [hunk 2]
```diff
@@ -82,139 +100,163 @@ After a successful OAP integration, you can use OAP SQL DDL to manage table inde
...
+   ndctl create-namespace -m fsdax -r region0
+   ndctl create-namespace -m fsdax -r region1
+   // show the created namespaces
+   fdisk -l
+   // create and mount file system
+   mount -o dax /dev/pmem0 /mnt/pmem0
+   mount -o dax /dev/pmem1 /mnt/pmem1
+   ```
+
+   In thi case file systems are generated for 2 numa nodes, which can be checked by "numactl --hardware". For a different number of numa nodes, a corresponding number of namespaces should be created to assure correct file system paths mapping to numa nodes.
```
Comment (HongW2019) [inline docs/OAP-User-Guide.md:220]: A little typo here "In this case, "

**Comment by zhixingheyi-tian** [comment]
Thanks Kevin!

**Pull request merged**